src/blocks/Media/schema.ts

```diff
@@ -0,0 +1,35 @@
+import {validators} from '@gravity-ui/page-constructor';
+
+import {BlockType} from '../../models/common';
+import {BlockBase, BlogBlockBase} from '../../schema/common';
+
+const {
+    common: {DataLensProps, VideoProps},
+    components: {ImageProps},
+} = validators;
+
+export const Media = {
+    [BlockType.Media]: {
+        type: 'object',
+        additionalProperties: false,
+        properties: {
+            ...BlockBase,
+            ...BlogBlockBase,
+            text: {
```

Comment:
Can we use Media schema from page-constructor here instead of writing it again?

Reply:
fix